### PR TITLE
Zabbix pack

### PR DESCRIPTION
**Pull request by psychopenguin**

### CODE REVIEW

**Comment by dzimine** [comment]
This is awesome! Looking forward for a sensor; if you follow nagios/sensu way of setting up the events, please add Zabbix specific instructions in the README.

File: packs/zabbix/actions/lib/base.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from pyzabbix import ZabbixAPI
+
+__all__ = [
+    'BaseZabbixAction'
+]
+
+
+class Action(object):
```
Comment (manasdk) [inline packs/zabbix/actions/lib/base.py]: Should import base Action from st2common like https://github.com/StackStorm/st2contrib/blob/master/packs/aws/actions/lib/action.py#L9.

**Comment by LindsayHill** [comment]
Could you please submit this against the [st2contrib repo?](https://github.com/StackStorm/st2contrib)

**Comment by Kami** [comment]
@psychopenguin Yeah, what Lindsday has said - It would be great to get this merged into st2contrib.

**Comment by psychopenguin** [comment]
Hi guys,

It has been a time I'm not playing around with this and I haven't made much progress on it (i don't even know if this pack is working with current version of ST2), but I'd be more than happy to PR this pack with st2contrib.

Let me know if this is still valid and I'll create a PR.

**Comment by LindsayHill** [comment]
I'm pretty sure it will still be valid. There's been some changes to
config.yaml, but the older methods are still supported.

So please do open a PR against st2contrib. We can always modify it.

**Pull request closed without merging**